### DISCUSSION TITLE
运动轨迹叠加在地图上产生了整体偏移，请问该如何修正？

### DISCUSSION THREAD

**Discussion by Epiphany-git** [yihong0618/running_page#289] [Q&A] [answered]
![Clipboard](https://user-images.githubusercontent.com/40231287/180735844-7a3c6506-2087-4af4-b232-0ac5cafa2c8c.png)

**Comment by yihong0618** [comment, marked as answer]
参考这个 issue
https://github.com/yihong0618/running_page/issues/198

**Reply by Epiphany-git** [reply]
谢谢，整体偏移的问题解决了。


不过又出现了另一个问题。
如下图，相同的一条记录，相比keep，展示的轨迹出现很多地方的跳点。请问这又是哪里出了问题呢？
![image](https://user-images.githubusercontent.com/40231287/180742723-8dee538a-0d2a-495a-b546-5b4ba7adff07.png)

**Reply by yihong0618** [reply]
no idea....

**Reply by Epiphany-git** [reply]
这段轨迹当时记录的时候曾经出现过GPS不稳定，比如keep图右边的那段直线其实也是不对的。不过keep展示出的整体形状是接近的。难道原始数据记录时本身就有很多噪点，keep展示时做了一些处理？

**Reply by yihong0618** [reply]
对的，keep 做了处理。

**Reply by Epiphany-git** [reply]
sad...谢谢，很棒的项目！

**Reply by yihong0618** [reply]
客气了。
